src/main/java/org/team1540/robot2024/commands/shooter/ShootSequence.java

```diff
@@ -1,17 +1,27 @@
 package org.team1540.robot2024.commands.shooter;
 
-import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.Commands;
+import edu.wpi.first.wpilibj2.command.FunctionalCommand;
 import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import org.team1540.robot2024.commands.indexer.PrepareFeederForShooter;
 import org.team1540.robot2024.subsystems.indexer.Indexer;
 import org.team1540.robot2024.subsystems.shooter.Shooter;
 
 public class ShootSequence extends SequentialCommandGroup {
     public ShootSequence(Shooter shooter, Indexer indexer) {
         addCommands(
                 Commands.parallel(
-                        indexer.feedToShooter(),
-                        new PrepareShooterCommand(shooter)
+                        new PrepareShooterCommand(shooter),
+                        Commands.sequence(
+                                new FunctionalCommand(
```

Comment:
yeah I suppose this works, I was thinking more along the lines of a factory method (return this from something like `indexer.commandNoteBackout()`) but I suppose this is fine